nixos/tests/microsoft-edge.nix

```diff
@@ -0,0 +1,41 @@
+{ system ? builtins.currentSystem
+, config ? {}
+, pkgs ? import ../.. { inherit system config; }
+, channelMap ? { # Maps "channels" to packages
+    stable        = pkgs.microsoft-edge;
+    beta          = pkgs.microsoft-edge-beta;
+    dev           = pkgs.microsoft-edge-dev;
+  }
+}:
+with import ../lib/testing-python.nix { inherit system pkgs; };
+with pkgs.lib;
```

Comment:
I'd try to avoid to bring too many attributes into the full scope (see https://nix.dev/recipes/best-practices#with-scopes) and instead either use full references or explicitly inherit the ones you need into the scope.